### ISSUE TITLE
npm package three versus three.js --> go for three.js

### ISSUE THREAD

**Issue by mattrei** [superguigui/Wagner#1]
Hi,

thanks for your package. ThreeJs needs modularity and this package helps a bit.

However I was struggling with a nasty error for quite some time, because my project dependet on 'three.js' and this project pulls in 'three' as a dependency. When those packages mismatch you get a hard to track down errors.

It only works when my project also uses 'three' as a depedency. Thats not a big deal for me, however I suggest that this project also changes its dependency to 'three.js' cause this is the package that gets published by _the_ mrdoob.

If you don't like the idea, please mention it in the README at least.

Thanks,
Matthias

**Comment by superguigui** [comment]
Hi Mattrei,
sorry for the very long response !
Indeed I knew Mrdoob was hoping to get the three.js repository back but I didn't know it happened. So as you suggested I switched from `three` unofficial dependency to official `three.js`.

One of the reason I did not reply right away is that I was thinking of doing something like in https://github.com/mattdesl/three-orbit-controls where THREE is not included as a direct dependency but passed as a parameter to the module. This would allow the user to chose whatever three version he'd want and avoid potential version conflicts.

This project however is a bit more complex and something like this might not be adapted... I don't know... I'm still thinking about it and I'm open to suggestions :)
